lighthouse-core/report/html/renderer/report-renderer.js

```diff
@@ -238,11 +223,25 @@ class ReportRenderer {
       const stickyHeader = this._dom.createElement('div', 'lh-sticky-header');
       stickyHeader.append(
         ...this._renderScoreGauges(report, categoryRenderer, specificCategoryRenderers));
-      reportFragment.appendChild(stickyHeader);
+      container.appendChild(stickyHeader);
     }
 
-    reportFragment.appendChild(headerContainer);
+    const categories = reportSection.appendChild(this._dom.createElement('div', 'lh-categories'));
+    for (const category of Object.values(report.categories)) {
+      const renderer = specificCategoryRenderers[category.id] || categoryRenderer;
+      // .lh-category-wrapper is full-width and provides horizontal rules between categories.
+      // .lh-category within has the max-width: var(--report-width);
+      const wrapper = renderer.dom.createChildOf(categories, 'div', 'lh-category-wrapper');
+      wrapper.appendChild(renderer.render(category, report.categoryGroups));
+    }
+
+    const reportFragment = this._dom.createFragment();
+    const topbarDocumentFragment = this._renderReportTopbar(report);
+
+    reportFragment.appendChild(topbarDocumentFragment);
     reportFragment.appendChild(container);
+    container.appendChild(headerContainer);
```

Comment:
these two lines are the only other real significant changes? `reportFragment` -> `rootContainer`?

Reply:
yes. most other stuff moved b/c of variable dependency. one thing moved because it made since to do as setup for the function (the cat renderer stuff)